indico/modules/events/abstracts/util.py

```diff
@@ -136,3 +193,41 @@ def render_list(self):
             'filtering_enabled': filtering_enabled,
             'filter_statistics': filter_statistics
         }
+
+
+def serialize_abstract_person_link(person_link):
```

Comment:
Hm, is it really worth it to create a new file for just one function? I would totally be in to create the file if we had a couple functions, but it's just one. If the majority thinks it's cleaner though, I'll gladly do it!